### ISSUE TITLE
Can't install on Python 3.5.0 because of enum34

### ISSUE THREAD

**Issue by chrisberkhout** [sdementen/piecash#8]
There is a [problem installing `enum34` on on Python 3.5.0](https://bitbucket.org/stoneleaf/enum34/issues/5/enum34-incompatible-with-python-35). It seems no longer necessary anyway, but since Piecash specifies it as a dependency I couldn't successfully `pip install piecash`.

I worked around that by getting the source, removing the dependency...

```diff
diff --git a/setup.py b/setup.py
index e795f2f..b3f27e1 100644
--- a/setup.py
+++ b/setup.py
@@ -261,7 +261,6 @@ setup_dict = dict(
     install_requires=[
                          'SQLAlchemy<=0.9.8',
                          'SQLAlchemy-Utils<=0.28.2',
-                         'enum34',
                          'pytz',
                          'tzlocal',
                          'future',
```

...and installing with `python setup.py install`.

I wonder if there is a way to fix this that doesn't cause problems for Python < 3.5.0 users?

**Comment by sdementen** [comment]
Good question. I just fell on the page https://bitbucket.org/stoneleaf/enum34/issues/5/enum34-incompatible-with-python-35 describing the issue on the enum34 repo but the issue is still open.

**Comment by sdementen** [comment]
I have just released 0.8.3 with the fix to install enum34 only on python <3.4.
I have also updated sqlalchemy to 0.9.10 and sqlalchemy-utils to 0.30.0.
thanks for reporting the issue !

**Comment by chrisberkhout** [comment]
Cool. `pip install` is working for me again. Thank you!
